examples/datasets/plot_limo_data.py

```diff
@@ -198,35 +192,20 @@
 
 ###############################################################################
 # As expected, the difference between Face A and Face B are very small.
-
-###############################################################################
+#
+#
 # Visualize effect of stimulus phase-coherence
 # --------------------------------------------
 #
-# For visualization purposes, we can transform the phase-coherence
-# variable back to fit roughly to it's original scale. Phase-coherence
-# determined whether a face stimulus could be identified as such. Thus,
+# Since phase-coherence
+# determined whether a face stimulus could be easily identified,
 # one could expect that faces with high phase-coherence should evoke stronger
 # activation patterns along occipital electrodes.
-#
-# As phase-coherence variable is a continuous variable, we'll need to
-# split the values of face coherence to percentiles to match roughly their
-# original scale (i.e, 0 to 100 % signal-to-noise ratio in 10% steps).
-# **Note:** We'll only do this for visualization purposes,
```

Comment:
What is the rationale for doing this? The data has 18 levels of standardized phase-coherence, and you're forcing it into 11 bins.

It's also somewhat distracting from the main point of the tutorial, which is regression.

Reply:
So as described in the main PR, it's actually 18 levels of phase-coherence, (0 to 85% with increments of 5%) not 11. This is because the data was (probably) taken from a sub-sample of the original paper, which (for whatever reasons) saw a slightly different version of the paradigm. Sorry that I didn't noticed that earlier on. I just assumed everybody saw the version of the paradigm which was cited on the original paper.

Reply:
Also I though, I might be useful for people to see what the effect of phase-coherence look like, at least on a descriptive level, and thus know what to expect from the linear regression analysis.